core/audits/third-party-summary.js

```diff
@@ -252,6 +252,9 @@ class ThirdPartySummary extends Audit {
         timeInMs: overallSummary.wastedMs,
       }),
       details,
+      metricSavings: {
+        TBT: overallSummary.wastedMs,
```

Comment:
see the comment on L110. we would need to bound the tasks we sum up by FCP and TTI, otherwise we risk overestimating the impact here

Reply:
Hmm that might explain https://github.com/GoogleChrome/lighthouse/issues/15004

Reply:
We should definitely do this, but the work required is more complicated because we need to get the TBT relevant tasks *after* lantern throttling and such.

I will find a different example that works as a simple impact calculation.